### PR TITLE
Separate remote and local files

### PR DESCRIPTION
**Pull request by seallard**
## Description
Closes https://github.com/Clinical-Genomics/housekeeper/issues/195.

Extracted some of the existing logic into a `FileService` and a `FileReportService`.

### Fixed
- Separate file tables for remote and local files

### CODE REVIEW

File: housekeeper/cli/get.py [hunk 1]
```diff
@@ -143,17 +144,18 @@ def files_cmd(
...
+
+    output_service.verbose = verbose
+    output_service.compact = compact
+    output_service.json = json
+
+    local = file_service.get_local_files(bundle=bundle, tags=tag_names, version_id=version_id)
+    remote = file_service.get_remote_files(bundle=bundle, tags=tag_names, version_id=version_id)
+
+    output_service.log_file_table(files=local, header="Local files")
+    output_service.log_file_table(files=remote, header="Remote files")
```
Comment (seallard) [inline housekeeper/cli/get.py:158]: These 4 lines could be wrapped in a single method in a service, maybe the `FileService`.

**Comment by seallard** [comment]
Works in stage

**Review by Vince-janv — APPROVED** [review]
Nice! Great that you restructured it as well 🙏

File: tests/cli/get/test_get_files.py [hunk 2]
```diff
@@ -3,127 +3,121 @@
...
-        cli_runner.invoke(files_cmd, ["--json"], obj=base_context).output
-    )
+    output = cli_runner.invoke(files_cmd, ["--json"], obj=base_context).output
 
-    # THEN assert that we get a list
-    assert isinstance(json_bundles, list)
-    # THEN assert that no files where fetched
-    assert len(json_bundles) == 0
+    # THEN assert that output was produced
+    assert output
```
[the remark this answers is not in the document]
Comment (seallard) [reply tests/cli/get/test_get_files.py]: Fixed in https://github.com/Clinical-Genomics/housekeeper/pull/196/commits/6999c108881511746150e0623b2776ce5ccc46a5

**Pull request merged**